src/test/nginx/nginx.conf

```diff
@@ -0,0 +1,56 @@
+server {
+    listen       8080;
+    server_name  localhost;
+
+    #charset koi8-r;
+    #access_log  /var/log/nginx/host.access.log  main;
+
+    location / {
+    	proxy_pass http://influxdb:8086/;
+        #root   /var/www/htmlllll;
+        #index  index.html index.htm;
+    }
+
+    location /influx-api/ {
+            proxy_pass http://influxdb:8086/;
+            proxy_http_version 1.1;
+            proxy_set_header Host $http_host;
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_redirect off;
+            error_log /tmp/inluxproxy.debug.log debug;
+   }
+
+
+    #error_page  404              /404.html;
+
+    # redirect server error pages to the static page /50x.html
+    #
+    error_page   500 502 503 504  /50x.html;
+    location = /50x.html {
+        root   /usr/share/nginx/html;
+    }
+
+    # proxy the PHP scripts to Apache listening on 127.0.0.1:80
```

Comment:
A lot of unrelated and commented code ?

Reply:
@majst01, I am going to help Mr. @vicctor if he has no time for this PR, just trying to make this PR able to merge. I will address your comment soon